pywebpush/__init__.py

```diff
@@ -473,7 +474,7 @@ def webpush(subscription_info,
             print("\t headers: {}".format(vapid_headers))
         headers.update(vapid_headers)
 
-    response = WebPusher(subscription_info, verbose=verbose).send(
+    response = WebPusher(subscription_info, requests_session=requests_session, verbose=verbose).send(
```

Comment:
Looks like the length of this line is causing CI to fail:

```
$ flake8 pywebpush
pywebpush/__init__.py:477:80: E501 line too long (101 > 79 characters)
The command "flake8 pywebpush" exited with 1.
```

One possible solution would be to apply Black's suggested formatting of:

```
    response = WebPusher(
        subscription_info, requests_session=requests_session, verbose=verbose
    ).send(
```